CCT_cold/config.json

```diff
@@ -0,0 +1,25 @@
+[
+    {
+        "name": "Columbia Card Task Cold Version",
+        "run": [
+                 "experiment.js",
+                 "style.css"
+               ],
+        "tag": "columbia_card_task_cold",
```

Comment:
The experiment "tag" must be equivalent to the folder name, which must be equivalent to the exp_id in Javascript. Currently we have:

- experiment tag: "columbia_card_task_cold"
- folder name: "CCT_cold"
- paths embedded in script: "CCT_cold"
- exp_id variable: "CCT"

all of these things must be the same, I would recommend changing to the first as it is more descriptive, and follows the standard of lowercase without special characters. The experiment will not validate in its current form.